src/libs/ReportUtils.ts

```diff
@@ -8472,20 +8472,41 @@ function isExported(reportActions: OnyxEntry<ReportActions>) {
 
 function getApprovalChain(policy: OnyxEntry<Policy>, expenseReport: OnyxEntry<Report>): string[] {
     const approvalChain: string[] = [];
+    const fullApprovalChain: string[] = [];
     const reportTotal = expenseReport?.total ?? 0;
+    const submitterEmail = PersonalDetailsUtils.getLoginsByAccountIDs([expenseReport?.ownerAccountID ?? -1]).at(0) ?? '';
 
-    // If the policy is not on advanced approval mode, we should not use the approval chain even if it exists.
-    if (!PolicyUtils.isControlOnAdvancedApprovalMode(policy)) {
+    if (PolicyUtils.isSubmitAndClose(policy)) {
```

Comment:
@ntdiary Clarify a bit, we need to return early if the approval mode is `OPTIONAL` then when the user submits the report, backend will not return error about the `managerAccountID` is incorrect.

Reply:
You can test with the case the step-up is the same with the original step except the approval mode is `OPTIONAL`. Then `no.24` creates an expense with tag/category match with rule and submit the report.

Reply:
Ooh ok I will test that out in OldDot now to see what happens!

Reply:
OK I just tested in OldDot and found that Category Approvers **are NOT** supported on Control, Submit & Close policies 🙏  - I honestly had never tested that, but glad it's tested now & it sounds like that's working in this PR 🙏

Reply:
> Interesting, @nkdengineer, so you tried this case:
> 
> 1. enable `Advanced Approval` mode
> 2. set category&tag rule approvers
> 3. then switch to `Submit and Close` mode directly.

FYI I don't think it's important to run the test in this order - a.k.a. it's not important to enable advanced approval first, then switch to submit & close... Right?!?